### DISCUSSION TITLE
Why can't see any tracking about RabbitMQ client?

### DISCUSSION THREAD

**Discussion by zengande** [microsoft/aspire#6645] [Q&A] [answered]
I installed the `Aspire.RabbitMQ.Client` package and added the source. But I cannot see any trace of RabbitMQ client. 
``` csharp
builder.Services.AddOpenTelemetry()
    .WithTracing(tracing =>
    {
         tracing.AddSource("Aspire.RabbitMQ.Client");
     });
```

May I ask if I have missed some configurations

**Comment by eerhardt** [comment, marked as answer]
The `Aspire.RabbitMQ.Client` package uses version 6 of RabbitMQ.Client, which doesn't have any tracing support. See [Enable Tracing for RabbitMQ Component when we move to their v7 (dotnet/aspire#562)](https://github.com/dotnet/aspire/issues/562).

Version 7 just shipped but we haven't had a chance to update yet. See [RabbitMQ.Client v7 breaks Aspire.RabbitMQ.Client component (dotnet/aspire#3956)](https://github.com/dotnet/aspire/issues/3956).
